### ISSUE TITLE
livesync not working well on emulator on some cases

### ISSUE THREAD

**Issue by ryc16** [NativeScript/NativeScript#2562]
I am facing strange behavior on livesync on emulator only. UI not update only changing xml file (do update on changing script file). Anyone faces the same. Is it a bug on NativeScript? Below is the details step. 

UI not updated while changing xml file as follow
#1. Plugged in ONLY emulator. 
#2. Changed xml file.
Even console saying 
```
Project successfully prepared
Transferring project files...
Successfully transferred main.xml.
Applying changes...
Successfully synced application com.myapps on device emulator-5554.
```

UI do updated as follow.
#1. Plugged in ONLY device. OR Plugged in both emulator and then device.
#2. Changed xml file.

This is on window
```
All NativeScript components versions information
┌──────────────────┬─────────────────┬────────────────┬───────────────┐
│ Component        │ Current version │ Latest version │ Information   │
│ nativescript     │ 2.1.1           │ 2.1.1          │ Up to date    │
│ tns-core-modules │ 2.1.0           │ 2.1.0          │ Up to date    │
│ tns-android      │ 2.1.1           │ 2.1.1          │ Up to date    │
│ tns-ios          │                 │ 2.1.1          │ Not installed │
└──────────────────┴─────────────────┴────────────────┴───────────────┘
```

Any idea? Or any way to have low level logging to see any exception or expected error.

**Comment by edelvalle** [comment]
Same problem here.

**Comment by harlankoehn** [comment]
I'm pretty sure this is an issue that has been fixed but the fix has not been released yet. I think it's in the @next build. There are a couple different issues (now closed) that already covered this problem. It was happening with XML/HTML and I think in some cases CSS files too. Here's the main issue I found on it: https://github.com/NativeScript/nativescript-cli/issues/1872

I think NativeScript 2.2 should be out any day now and should include the fix for this.
